ansible/roles/schulcloud-server-h5p-proxy/templates/api-h5p-proxy-deployment.yml.j2

```diff
@@ -0,0 +1,72 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: api-h5p-proxy-deployment
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: api-h5p-proxy
+    app.kubernetes.io/part-of: schulcloud-verbund
+    app.kubernetes.io/version: "{{ H5P_PROXY_IMAGE_TAG }}"
+    app.kubernetes.io/name: api-h5p-proxy
+    app.kubernetes.io/component: api-h5p
+    app.kubernetes.io/managed-by: ansible
+    git.branch: {{ DOF_APP_DEPLOY_BRANCH_NAME }}
+    git.repo: {{ DOF_APP_DEPLOY_REPO_NAME }}
+spec:
+  replicas: {{ H5P_PROXY_REPLICAS|default(1, true) }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 1
+  selector:
+    matchLabels:
+      app: api-h5p-proxy
+  template:
+    metadata:
+      labels:
+        app: api-h5p-proxy
+        app.kubernetes.io/part-of: schulcloud-verbund
+        app.kubernetes.io/version: "{{ H5P_PROXY_IMAGE_TAG }}"
+        app.kubernetes.io/name: api-h5p-proxy
+        app.kubernetes.io/component: api-h5p
```

Comment:
The componet shoud be only h5p like all other h5p things.

Reply:
Ack